### PR TITLE
Fix stars

### LINKED ISSUE #16: Fix star highlighting issue on onboard-dashboard screen
When hovering the star SVGis filled in with black but not when the cursor is hovering the path or the center of the star

### PR DESCRIPTION
**Pull request by zachjamesgreen**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Build is passing
## Pull request type
Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):
## What is the current behavior?
Issue Number: 
closes #16 
## What is the new behavior?
Fix star flickering issue by using PNGs instead of SVGs

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/components/onboard.js [hunk 1]
```diff
@@ -3,18 +3,41 @@ import Stars from './stars.js'
...
+      },
+    ]
     return(
       <div className="font-playfair">
         <h1 className="text-center mb-12 text-3xl">To give you the best recommendations, please rate these cocktails</h1>
         <div className="flex justify-center space-x-8">
-        {Array.from({length: 5 }, (v, i) => (
-            <div className="text-center" key={i}>
-              <img src={`${image}?${i}`} alt="Cocktail" />
+        {data.map((cocktail) => {
```
Comment (markcyen) [inline src/components/onboard.js:32]: functions!!! ...very functional. this and that... ⍩

File: src/components/star.js [hunk 2]
```diff
@@ -7,13 +7,13 @@ export default class Star extends React.Component {
   }
 
   render() {
-    return(
-        <svg 
-          data-star-id={this.props.starId}
-          className={`${this.props.marked ? 'text-black' : ''} hover:fill-current transition`}
-          width="26" height="23" viewBox="0 0 26 23" fill={this.props.marked ? 'black' : 'none'} xmlns="http://www.w3.org/2000/svg">
-          <path d="M12.8567 3.15498L14.7947 8.95425L15.023 9.63729H15.7431H21.9352L16.9501 13.1587L16.3431 13.5875L16.5787 14.2924L18.4978 20.0351L13.4336 16.4578L12.8567 16.0502L12.2797 16.4578L7.21548 20.0351L9.13464 14.2924L9.37021 13.5875L8.76316 13.1587L3.77813 9.63729H9.97015H10.6903L10.9186 8.95425L12.8567 3.15498Z" stroke="black" strokeWidth="2"/>
```
Comment (markcyen) [inline src/components/star.js:15]: what. the. ?!?! ௺

File: src/components/stars.js [hunk 3]
```diff
@@ -25,6 +31,19 @@ export default class Stars extends React.Component {
   handleClick = (e) => {
     let val = e.target.getAttribute('data-star-id') || this.state.rating
     this.setState({rating: val})
+    if (val != null && !isNaN(val)) {
```
Comment (markcyen) [inline src/components/stars.js:34]: I can understand this... 🤔

**Review by markcyen — APPROVED** [review]
Ok!

File: src/components/onboard.js [hunk 4]
```diff
@@ -3,18 +3,41 @@ import Stars from './stars.js'
...
+          return (
+            <div className="text-center" key={cocktail.name}>
+              <img src={cocktail.image} alt="Cocktail" />
               <p className="mt-4">Cocktail Name</p>
-              <Stars cname="stars flex mt-4 justify-center"/>
-          </div>
-          ))}
+              <Stars cname="stars flex mt-4 justify-center" name={cocktail.name}/>
+            </div>
+          )
```
Comment (mkrumholz) [inline src/components/onboard.js:39]: This looks way cleaner. Nice!

File: src/components/star.js [hunk 5]
```diff
@@ -7,13 +7,13 @@ export default class Star extends React.Component {
...
-        </svg>
-    )
+    if (this.props.marked) {
+      return <img data-star-id={this.props.starId} src="star_marked.png" alt="marked star" />
+    } else {
+      return <img data-star-id={this.props.starId} src="star_not_marked.png" alt="empty star" />
+    }
```
Comment (mkrumholz) [inline src/components/star.js:14]: These are working great now. 
Screen Shot 2021-08-04 at 9 35 53 AM

File: src/components/stars.js [hunk 6]
```diff
@@ -6,10 +7,15 @@ export default class Stars extends React.Component {
...
       rating: parseInt(this.props.value),
+      name: this.props.name,
       selection: 0
     }
   }
 
+  getReccs = () => {
+    // axios('http://localhost:8080/')
+  }
+
```
Comment (mkrumholz) [inline src/components/stars.js:18]: This endpoint should be ready to go for you

File: src/components/stars.js [hunk 7]
```diff
@@ -25,6 +31,19 @@ export default class Stars extends React.Component {
   handleClick = (e) => {
     let val = e.target.getAttribute('data-star-id') || this.state.rating
     this.setState({rating: val})
+    if (val != null && !isNaN(val)) {
+      let data = {
+        user_id: 1,
+        stars: val,
+        cocktail_id: 11007
```
Comment (mkrumholz) [inline src/components/stars.js:38]: We'll need to update this, right? So it's not hardcoded in?

**Review by mkrumholz — APPROVED** [review]
Looks like there is some data hardcoded that we'll need to update later, but I tested the stars on local and it works way better. Nice work!

**Pull request merged**